scipy/special/setup.py

```diff
@@ -13,6 +13,20 @@
                      (numpy.__version__, numpy.__file__)) from e
 
 
+def cxx_pre_build_hook(build_ext, ext):
+    from scipy._build_utils.compiler_helper import (get_cxx_std_flag,
+                                                    try_add_flag)
+    cc = build_ext._cxx_compiler
+    args = ext.extra_compile_args
+
+    std_flag = get_cxx_std_flag(cc)
+    if std_flag is not None:
+        args.append(std_flag)
+    if sys.platform == 'darwin':
+        args.append('-mmacosx-version-min=10.7')
```

Comment:
min version is `10.9` now. I think for maintainability this flag should be taken from `set_cxx_flags_hook` in scipy._build_utils.compiler_helper`.

Reply:
@rgommers, do you mean that the function `scipy._build_utils.compiler_helper.set_cxx_flags_look` can simply replace the `cxx_pre_build_hook` function defined in file `setup.py` here?

Reply:
Yes exactly. Sorry that comment was unclear. It's almost an exact copy now.

Reply:
Thanks. In f2e1b07f I did exactly that. (also it was `set_cxx_flags_hook` not `set_cxx_flags_look`, sorry about the typo!)